site/_data/docs/lighthouse/toc.yml

```diff
@@ -1,13 +1,16 @@
 - url: /docs/lighthouse/overview/
-- url: /docs/lighthouse/seo/
```

Comment:
rearranged this so that the TOC is in the same order as the Lighthouse report (perf, a11y, best practices, seo, pwa)